docs/api.rst

```diff
@@ -64,12 +64,24 @@ API reference
 .. autoclass:: appsettings.CallablePathSetting
     :members:
 
+``appsettings.NestedDictSetting`` setting
+-----------------------------------------
+
+.. autoclass:: appsettings.NestedDictSetting
+    :members:
+
 ``appsettings.NestedSetting`` setting
```

Comment:
`NestedSetting` should be marked as deprecated in the docs as well.

Reply:
Fixed.